packages/framework/src/Model/Customer/CustomerUserFacade.php

```diff
@@ -49,31 +49,31 @@ class CustomerFacade
     protected $userFactory;
 
     /**
-     * @var \Shopsys\FrameworkBundle\Model\Customer\CustomerPasswordFacade
+     * @var \Shopsys\FrameworkBundle\Model\Customer\CustomerUserPasswordFacade
      */
     protected $customerPasswordFacade;
```

Comment:
When you rename class, you should also rename all variables that store such class